spec/rails_test_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_spec_helper'
+
+# Rails5 doesn't work with Ruby 3.x
+def default_rails_versions
+  # TODO: consider testing other versions of Rails
+  # testing_ruby_2? ? [ 5, 6 ] : [ 6 ]
+  [ 7 ]
```

Comment:
Is this something that will be changed back before merge?

Reply:
I updated this so 5, 6, 7 will all be run. The test only checks that `APPMAP=true bundle exec rake` succeeds, though, which is enough to show that the fix works. We can follow up with some more tests.